client/src/extension.ts

```diff
@@ -165,7 +159,7 @@ function getPreviewContent(content: String) : string {
 	return `
 	<!DOCTYPE html>
 	<body>
-		<script src="https://cdnjs.cloudflare.com/ajax/libs/d3/5.16.0/d3.min.js"></script>
+		<script src="https://unpkg.com/d3@5.16.0/dist/d3.min.js"></script>
 		<script src="https://unpkg.com/@hpcc-js/wasm@0.3.11/dist/index.min.js"></script>
 		<script src="https://unpkg.com/d3-graphviz@3.0.5/build/d3-graphviz.min.js"></script>
```

Comment:
Like the move off of [cdnjs](https://cdnjs.com/) since it's [curated and missing libraries](https://github.com/cdnjs/cdnjs/blob/master/CONTRIBUTING.md#b-request-a-new-library)
We should add [Subresource Integrity](https://en.wikipedia.org/wiki/Subresource_Integrity):
```suggestion
		<script src="https://cdn.jsdelivr.net/npm/d3@5.16.0/dist/d3.min.js" integrity="sha256-Xb6SSzhH3wEPC4Vy3W70Lqh9Y3Du/3KxPqI2JHQSpTw=" crossorigin="anonymous"></script>
		<script src="https://cdn.jsdelivr.net/npm/@hpcc-js/wasm@0.3.11/dist/index.min.js" integrity="sha256-ed1KzxE+MS5A4K5j2j73fLv/1MNRWJ73e8N0V/mU/hU=" crossorigin="anonymous"></script>
		<script src="https://cdn.jsdelivr.net/npm/d3-graphviz@3.0.5/build/d3-graphviz.min.js" integrity="sha256-dZ4Que6P88a308S9XMJrqbcD7iEjcaejEPLGkEJrEZQ=" crossorigin="anonymous"></script>
```
https://www.jsdelivr.com/package/npm/d3?version=5.16.0
https://www.jsdelivr.com/package/npm/@hpcc-js/wasm?version=0.3.11
https://www.jsdelivr.com/package/npm/d3-graphviz?version=3.0.5

Reply:
Done. We're also supposed to [add a content security policy](https://code.visualstudio.com/api/extension-guides/webview#content-security-policy), but I couldn't get this to work